linera-service/src/linera/main.rs

```diff
@@ -1118,6 +1119,7 @@ impl Runnable for Job {
                 println!("{}", outcome.chain_id);
                 println!("{}", outcome.message_id);
                 println!("{}", outcome.certificate_hash);
+                println!("{}", owner);
```

Comment:
Just by looking at these `println`s it's impossible to tell "what is what". Is it so b/c this format is easier to consume programmatically?